modules/rds/variables.tf

```diff
@@ -36,21 +36,25 @@ variable "config_path" {
 variable "rds_database_name" {
   type        = string
   description = "Database name"
+  default     = ""
```

Comment:
is it possible to create RDS instance with a blank database name?

Reply:
added default value "exampledb"